app/bundles/PageBundle/Form/Type/DashboardHitsInTimeWidgetType.php

```diff
@@ -25,7 +26,7 @@ class DashboardHitsInTimeWidgetType extends AbstractType
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('flag', 'choice', [
+        $builder->add('flag', ChoiceType::class, [
```

Comment:
All choice types need new `choices_as_values` option and the choices must flip labels and values. Fix that for all choice types in all bundles you've refactored so far, please.
See: 
https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#heading=h.imucd6w2ou47

Reply:
Done. TY!